sdk/Lusid.Sdk.Tests/tutorials/Instruments/DemoEquityOption.cs

```diff
@@ -67,76 +34,47 @@ public void DemoEquityOptionCreation()
-                _testDataUtilities.CreateAndUpsertConstantVolSurface(scope, _demoEffectiveAt, option, model, 0.2m);
+                CreateAndUpsertConstantVolSurface(scope, TestDataUtilities.EffectiveAt, option, model, 0.2m);
             if (model == ModelSelection.ModelEnum.Bachelier)
-                _testDataUtilities.CreateAndUpsertConstantVolSurface(scope, _demoEffectiveAt, option, model, 10m);
+                CreateAndUpsertConstantVolSurface(scope, TestDataUtilities.EffectiveAt, option, model, 10m);
 
             // CREATE Black-Scholes recipe specifying where to look for market data and which metrics to return
             // if in a larger portfolio, we would make a specific VendorModelRule specifying that equity options are to be valued using Black-Scholes
-            var recipeCode = $"EquityOption_ValuationRecipe-{modelName}";
-            var pricingOptions = new PricingOptions(new ModelSelection(ModelSelection.LibraryEnum.Lusid, model));
-            var recipe = new ConfigurationRecipe
-            (
-                scope: scope,
-                code: recipeCode,
-                market: new MarketContext(
-                    new List<MarketDataKeyRule> {},
-                    options: new MarketOptions(defaultSupplier: "Lusid", defaultScope: scope, defaultInstrumentCodeType: "RIC")
-                    ),
-                pricing: new PricingContext(options: pricingOptions)
-            );
-            _testDataUtilities.UpsertRecipe(recipe);
-
-            // DEFINE the metrics that we wish to return
-            string ValuationDateKey = "Analytic/default/ValuationDate";
-            string InstrumentTag = "Analytic/default/InstrumentTag";
-            string HoldingPvKey = "Holding/default/PV";
-            var valuationSpec = new List<AggregateSpec>
-            {
-                new AggregateSpec(ValuationDateKey, AggregateSpec.OpEnum.Value),
-                new AggregateSpec(InstrumentTag, AggregateSpec.OpEnum.Value),
-                new AggregateSpec(HoldingPvKey, AggregateSpec.OpEnum.Value)
-            };
-
-            // CHOOSE valuation dates
-            var valuationSchedule = new ValuationSchedule(effectiveAt: _demoEffectiveAt);
-
-            // CONSTRUCT and PERFORM valuation request
-            var instruments = new List<WeightedInstrument> {new WeightedInstrument(1, "some-holding-identifier", option)};
-            var inlineValuationRequest = new InlineValuationRequest(
-                recipeId: new ResourceId(scope, recipeCode),
-                metrics: valuationSpec,
-                sort: new List<OrderBySpec> {new OrderBySpec(ValuationDateKey, OrderBySpec.SortOrderEnum.Ascending)},
-                valuationSchedule: valuationSchedule,
-                instruments: instruments);
-            var valuation = _aggregationApi.GetValuationOfWeightedInstruments(inlineValuationRequest);
+            var valuation = Valuation(option, scope, model, EffectiveAt, inlineValuationRequest);
```

Comment:
do the models all use the same recipe? I think for clarity we should move the recipe here so that new users have a template to work from

Reply:
They do but can pull out to make it explicit and provide the template.